src/components/OnThisPageNav.vue

```diff
@@ -20,7 +20,8 @@
           class="base-link"
           @click.native="handleFocusAndScroll(item.anchor)"
         >
-          <WordBreak>{{ item.title }}</WordBreak>
+          <WordBreak v-if="item.i18n">{{ $t(item.title) }}</WordBreak>
```

Comment:
Does this if statement actually check if the `title` is translated, or does it only check if the item has any translated keys?

Reply:
I'm a little confused about this as well. Shouldn't the `title` field already be localized in the JSON for this since it's a dynamic string and not a hardcoded one from DocC-Render? I'm not sure we would need to make any changes here if that's the case (which is how I'm expecting it to work).

Reply:
At `On This Page` we have a mix of hardcoded values and dynamic ones. Most of the titles are hardcoded and that's why `addOnThisPageSection` function has by default to `i18n` to `true`, but for the dynamic ones, I turn `i18n` into `false` because they should be translated by the backend. Example: 
```js
this.store.addOnThisPageSection({
        title,
        anchor: AppTopID,
        level: 1,
      }, { i18n: false });
```
on `src/mixins/onThisPageRegistrator.js`